plutus-playground-client/src/Types.purs

```diff
@@ -369,6 +386,7 @@ instance corecursiveSimpleArgument :: Corecursive SimpleArgument SimpleArgumentF
   embed (SimpleTupleF xs) = SimpleTuple xs
   embed (SimpleArrayF schema xs) = SimpleArray schema xs
   embed (SimpleObjectF schema xs) = SimpleObject schema xs
+  embed (ValueArgumentF schema x) = ValueArgument schema x
```

Comment:
off topic but I wonder if there could be a generic version of embed and project?

Reply:
It's definitely possible: http://hackage.haskell.org/package/recursion-schemes-5.1.2/docs/src/Data.Functor.Foldable.html#embed

Reply:
good 'ol uncle Ed